test/Evaluate/test_folding.sh

```diff
@@ -19,26 +19,21 @@
 #   - test_x is not folded (it is neither .true. nor .false.). This means the
 #     compiler could not fold the expression.
 
-PATH=/usr/bin:/bin
-srcdir=$(dirname $0)
-F18CC=${F18:-../../../tools/f18/bin/f18}
-CMD="$F18CC -fdebug-dump-symbols -fparse-only"
+CMD="$2 -fdebug-dump-symbols -fparse-only"
 
 if [[ $# < 1 ]]; then
-  echo "Usage: $0 <fortran-source> [-pgmath=<true/false>]"
+  echo "Usage: $0 <fortran-source> [<f18-executable>] <temp test dir>"
```

Comment:
Same comment as on common.sh. <f18-executable> is no longer an optional argument, so lose the [] notation.